rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -366,13 +340,26 @@ This task tree matches the one shown earlier for merge-sort.
 
 ## Open Questions in Design
 
-Some open questions that remain:
-
+Questions that should be addressed before moving into `experimental` and providing a preview feature:
 - Are the suggested APIs sufficient?
+- Are there additional use cases that should be considered that we missed in our analysis?
+- Semantics for `task_arena::enqueue` method that takes a `task_handle` object should be defined for all possible states of the handle.
```

Comment:
That's true not only for `enqueue` but also for the task group functions that take a handle.

Reply:
Rewritten and listed all usages of `task_handle` semantics for which should be considered.